lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -322,29 +332,43 @@ def test_get_repeat_keys(self, query_as):
             (["dataset.name:fio"], "datasets.name = 'fio'"),
             (
                 ["dataset.metalog.pbench.script:fio"],
-                "dataset_metadata.key = 'metalog' AND dataset_metadata.value[['pbench', 'script']] = 'fio'",
+                "dataset_metadata.key = 'metalog' "
+                "AND dataset_metadata.value[['pbench', 'script']] = 'fio'",
             ),
             (
                 ["user.d.f:1"],
-                "dataset_metadata.key = 'user' AND dataset_metadata.value[['d', 'f']] = '1' AND dataset_metadata.user_id = '3'",
+                "dataset_metadata.key = 'user' AND dataset_metadata.value[['d', "
+                "'f']] = '1' AND dataset_metadata.user_id = '3'",
             ),
             (
                 ["dataset.name:~fio", "^global.x:1", "^user.y:~yes"],
-                "(datasets.name LIKE '%' || 'fio' || '%') AND (dataset_metadata.key = 'global' AND "
-                "dataset_metadata.value[['x']] = '1' OR dataset_metadata.key = 'user' AND "
-                "((dataset_metadata.value[['y']]) LIKE '%' || 'yes' || '%') AND dataset_metadata.user_id = '3')",
+                "(datasets.name LIKE '%' || 'fio' || '%') AND "
+                "(dataset_metadata.key = 'global' AND "
+                "dataset_metadata.value[['x']] = '1' OR dataset_metadata.key "
+                "= 'user' AND ((dataset_metadata.value[['y']]) LIKE '%' || "
+                "'yes' || '%') AND dataset_metadata.user_id = '3')",
```

Comment:
If you make a change for some other reason, it would be nice if you broke these lines on `AND`'s or `OR`'s instead of in the middle of expressions.  (We can definitely afford to spread 340-341 across a third line, and it would be nicer if the breaks in 347-348 and 348-349 landed differently.)